components/Modal/Modal.unit.test.jsx

```diff
@@ -92,12 +92,34 @@ describe('<Modal />', () => {
       );
 
       const focusedElement = document.activeElement;
+
       expect(
         modal
           .find('button')
           .at(0)
           .getDOMNode(),
       ).toBe(focusedElement);
     });
+
+    it('should not focus first element when Modal is opened', () => {
+      mount(
+        <Modal>
+          <Modal.Content>
+            <p>
+              Some text<a href="catho.com">Some link</a>
+            </p>
+          </Modal.Content>
+          <Modal.Footer>
+            <button type="button">Cancel</button>
+            <button type="button">Ok</button>
+          </Modal.Footer>
+        </Modal>,
+      );
+
+      const focusedElement = document.activeElement;
+
+      expect(focusedElement.nodeName).not.toBe('A');
+      expect(focusedElement.nodeName).toBe('BUTTON');
```

Comment:
Question. Why two expects? Wouldn't just one already solve the job?

Reply:
I think it's necessary to evidence first item in focus can't be a link, but I notice description of test is wrong.